recipes/sbp/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+from textwrap import dedent
+
+import yaml
+
+from conans import ConanFile, CMake, tools
+
+
+class SbpConan(ConanFile):
+    name = "sbp"
+    license = "MIT"
+    author = "Swift Navigation <dev@swiftnav.com>"
+    url = "https://github.com/swift-nav/libsbp"
+    description = "Swift Binary Protocol client library"
+    topics = ("gnss",)
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": True, "fPIC": True}
+    generators = "cmake"
+    generators = "cmake"
+    exports = "submoduledata.yml"
+    exports_sources = "c"
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
```

Comment:
```suggestion
            del self.options.fPIC

    def configure(self):
        if self.options.shared:
            del self.options.fPIC

    def source(self):
```

Reply:
I did this change, and locally (on mac) I see
> [HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] OK. 'fPIC' option found and apparently well managed

but CI says
> [HOOK - conan-center.py] pre_build(): [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not found
> ..
> [HOOK - conan-center.py] pre_build(): ERROR: [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not managed correctly. Please remove it for shared option: del self.options.fpic (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H007)

Reply:
Windows needs the `fPIC` option removed.